src/crimson/osd/ops_executer.h

```diff
@@ -275,4 +265,23 @@ OpsExecuter::osd_op_errorator::future<> OpsExecuter::flush_changes(
   }
 }
 
+class PgOpsExecuter {
+  PG& pg;
```

Comment:
nit, might want to put the public parts earlier. see https://google.github.io/styleguide/cppguide.html#Declaration_Order